src/store/Account.js

```diff
@@ -737,12 +756,79 @@ const mutations = {
 
   [types.SET_WHITELIST_REQUEST] (state) {},
 
-  [types.SET_WHITELIST_SUCCESS] (state) {},
+  [types.SET_WHITELIST_SUCCESS] (state) { },
 
   [types.SET_WHITELIST_FAILURE] (state, err) {
     handleError(state, err)
   },
 
+  [types.SET_FEE_SUCCESS] () {},
```

Comment:
It is duplication of line 741, but overall no any function uses this types. I guess we can remove it